### ISSUE TITLE
Only ctrl + alt +d & ctrl + w are working after the patch

### ISSUE THREAD

**Issue by Mandarine16** [PoE-TradeMacro/POE-TradeMacro#546]
Hi,

I just updated the poe trade macro, and since then, only ctrl + alt + d and the ctrl + w hotkeys are working. Nothing happens when I try to use the other ones.

Also when using ctrl + alt + d I do get the window, but it make me go back to windows.

Can anyone help me figure out what is wrong ? I uninstalled and re-installed everything twice but still no change. 

Thanks

(trademacro v2.5.7)
Using Windows 7

**Comment by Mandarine16** [comment]
Okay so, I just realised that its apparently working, but somehow the small window that appears when pressing ctrl + d for exemple just show up BEHIND the game window (I only see it if I alt tab without moving mouse at all).

Any way I can fix that ?

**Comment by aRTy42** [comment]
Do you have your client in "fullscreen" mode, instead of "windowed" or "borderless window"?

**Comment by Mandarine16** [comment]
Yes I actually checked a few minutes ago and was about to make an update.
So my game was in "Fullscreen", I changed to "Windowed Fullscreen" and the tooltips are properly showing up.

What is really weird tho is that before (up to yesterday), the macro was working without any issue. I'm new to PoE and newer on PoEtrade macro, so I don't know if I had the game in "Windowed Fullscreen" before, which would explained why it worked, or if for some reason it worked in "Fullscreen" before the update.

Either way, it's very great to have it working again.

If anyone experiences the same issue, I suggest you make sure your window options didn't change for some reason !

This is not the place, but congratulation on making this great tool. PoE economy is very difficult to understand as a new player, and your tool is really helpful !
